### DISCUSSION TITLE
SLP for all jobs on `plotman status`?

### DISCUSSION THREAD

**Discussion by mcybz** [ericaltendorf/plotman#654] [Q&A]
Does this look normal to anyone?

![image](https://user-images.githubusercontent.com/83850054/119418906-144c7180-bcc7-11eb-963c-647d0b0b1204.png)

I did another `plotman status` 2 hours ago and same thing. I thought we were meant to see mix of SLP, DSK, RUN etc?

**Comment by pspardon** [comment]
This is normal, check the [plot statuses wiki page.](https://github.com/ericaltendorf/plotman/wiki/Plot-statuses)

**Comment by XerrorYT** [comment]
SLP seems to be the default state for any plot in phase 1.
As all of those plots are phase 1 there shouldnt be a problem.
